package.json

```diff
@@ -119,6 +119,7 @@
     "react-dom": "16.4.1",
     "react-perfect-scrollbar": "1.0.0",
     "react-router-dom": "4.2.2",
+    "react-transition-group": "^2.3.1",
```

Comment:
@morteza-binary , this will be a peer dependency with SmartCharts library. But I've discussed with bruce and we'll both be setting it to a fixed version.